Dart/README.md

```diff
@@ -43,7 +43,7 @@ Prerequisites:
 
 4. Open intellij-community project, compile it.
    Open File | Project Structure | Modules | `[+]` | Import Module, select intellij-plugins/Dart/Dart-community.iml.
-   In the same Project Structure dialog open the Dependencies tab of the `intellij.idea.community.main` module,
+   In the same Project Structure dialog open the Dependencies tab of the `intellij` > `idea.community` > `main` module,
```

Comment:
Strictly speaking, the name of the module is `intellij.idea.community.main`. Tree presentation can even be disabled in the context menu (Flatten Modules). How about 
> open the Dependencies tab of the `intellij.idea.community.main` module (`intellij` > `idea.community` > `main` node)

or something else, both correct and clear?

Reply:
Makes sense. I needed to do some digging in the tree structure to find the right module, so calling out both the name and the location would be useful